files/gitignore

```diff
@@ -1,29 +1,18 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# compiled output
-dist/
-declarations/
-
-# npm/pnpm/yarn pack output
-*.tgz
-
-# dependencies
-node_modules/
```

Comment:
Was `node_modules/` here removed by accident? Created a new v2 addon using the `main` branch and the `node_modules` folder is getting picked up by Git.